andela/mrm_api
# #161179706 Cancel event functionality

### What does this PR do?

- This PR enables a cancelled event to be captured and stored in the database and the event removed from the calendar.

#### How should this be manually tested?
- Run the server.
- Test the mutation at localhost:5000/mrm.
- Run `cancelledEvent` mutation with `calendarId` and `eventId`  as the arguments.
- The cancellation status should be updated to `true` while the checkin status should be updated to `false`

### What are the relevant pivotal tracker stories?

#161179706 


#### Screenshots
<img width="1429" alt="cancelled event" src="https://user-images.githubusercontent.com/6715848/47178955-5d750180-d325-11e8-9708-19945ff40e5a.png">
